apps/ledger-live-mobile/src/newArch/features/LandingPages/screens/LargeMoverLandingPage/Components/Card.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { ScrollView, StyleSheet } from "react-native";
+import { Flex } from "@ledgerhq/native-ui";
+import Svg, { Defs, LinearGradient, Rect, Stop } from "react-native-svg";
+import { Ticker } from "./Ticker";
+import { LargeMoverGraph } from "./LargeMoverGraph";
+import { TimeFrame } from "./TimeFrame";
+import { Performance } from "./Performance";
+import { PriceAndVariation } from "./PriceAndVariation";
+import { Footer } from "./Footer";
+import { CurrencyData, KeysPriceChange } from "@ledgerhq/live-common/market/utils/types";
+import getWindowDimensions from "~/logic/getWindowDimensions";
+import { Informations } from "./Information";
+import { useTheme } from "styled-components/native";
+import { getCryptoCurrencyById } from "@ledgerhq/live-common/currencies/index";
+
+const graphRangeMap: Record<KeysPriceChange, "24h" | "7d" | "1m" | "1y"> = {
+  [KeysPriceChange.hour]: "1y", // not used
+  [KeysPriceChange.day]: "24h",
+  [KeysPriceChange.week]: "7d",
+  [KeysPriceChange.month]: "1m",
+  [KeysPriceChange.year]: "1y",
+};
+
+type CardProps = CurrencyData & {
+  range: KeysPriceChange;
+  setRange: (range: KeysPriceChange) => void;
+};
+
+const { width } = getWindowDimensions();
+
+export const Card: React.FC<CardProps> = ({
+  id,
+  price,
+  priceChangePercentage,
+  chartData,
+  low24h,
+  high24h,
+  range,
+  setRange,
+  ticker,
+  marketcap,
+  totalVolume,
+  marketCapChangePercentage24h,
+  maxSupply,
+  circulatingSupply,
+  totalSupply,
+  ath,
+  atl,
+  athDate,
+  atlDate,
+}) => {
+  const graphWidth = width * 0.86;
+  const timeframehWidth = width * 0.96;
+  const { colors } = useTheme();
+  const middleColor = colors.neutral.c20;
+  const currency = getCryptoCurrencyById(id);
+
+  return (
+    <Flex backgroundColor="neutral.c20" borderRadius={20} marginRight={5} marginLeft={5}>
+      <ScrollView showsVerticalScrollIndicator={false}>
+        <Flex padding={4}>
+          <Flex padding={4}>
+            <Ticker currencyId={id} />
+          </Flex>
+          <PriceAndVariation
+            price={price}
+            priceChangePercentage={priceChangePercentage}
+            range={range}
+          />
+          <Flex alignItems="center">
+            <LargeMoverGraph
+              chartData={chartData}
+              range={graphRangeMap[range]}
```

Comment:
```suggestion
              range={range}
```

works no? because range is type of KeysPriceChange